src/Components/TodoList/ListItem/ListItem.tsx

```diff
@@ -1,9 +1,29 @@
-import React from 'react'
+import { Box, Button, Checkbox, Typography } from "@mui/material";
+import React from "react";
 
-export default function ListItem() {
+interface ListItemProps {
+  description: string;
+}
+
+export default function ListItem({ description }: ListItemProps) {
   return (
-    <div>
-      list item
-    </div>
-  )
+    <Box sx={{ display: "flex", flex: 1, alignItems: "center" }}>
+      <Checkbox />
+      <Typography sx={{ display: "flex", width: "100%" }}>
+        {description}
+      </Typography>
+      <Button
+        variant="contained"
+        sx={{
+          bgcolor: "red",
+          textTransform: "none",
+          "&:hover": {
+            backgroundColor: "#8c1c03",
```

Comment:
Colors in the application should not be literal values.
please move to colors.ts -> COLORS object.